create-quasar/index.js

```diff
@@ -102,6 +102,12 @@ async function run () {
       }
     })
 
+    await utils.extendJsonFile(scope.projectFolder + '/package.json', {
+      scripts: {
+        serve: "quasar serve",
```

Comment:
Let aside it won't serve anything if you don't provide a path for it to serve files from (`quasar serve dist/spa` would be a good default), are you really sure `quasar serve` will work as-is for all package managers?

I can already tell you that Yarn 1 will resolve the `quasar` reference there as the local quasar bin served from `@quasar/app-*` packages and break, since `serve` command is provided by the `@quasar/cli` the user is supposed to have installed globally.
For Yarn 1 this should work: `"serve": "$(yarn global bin)/quasar serve dist/spa"`

Pnpm could possibly have a similar problem

How did you test it was working correctly and on which package managers and versions?

Reply:
To provide different versions of the `serve` script you're supposed to check with which package manager the user is executing `create-quasar` package

You can deduce the used package manager by analyzing the process.env data

Taken from the example project I provided you access to:
```
export const runningPackageManager = (process.env.npm_config_user_agent
  ?.split(" ")[0]
  .split("/")[0]);
```

Notice that if the package manager cannot be determined due to the missing `npm_config_user_agent`, you shouldn't add the `serve` script